### ISSUE TITLE
[Camera Support]: Can't get any image

### ISSUE THREAD

**Issue by MalekBakary** [blakeblackshear/frigate#8937]
### Describe the problem you are having

I'm trying to get Frigate running on Home Assistent but can't get any image

### Version

0.12.1

### Frigate config file

```yaml
mqtt:
  host: 192.168.0.198
  port: 1883
  user: homeassistant
  password: *****

cameras:
  Tapo_Camera_Door:
    enabled: True
    ffmpeg:
      inputs:
        - path: rtsp://***:***@192.168.0.26/stream1
          roles:
            - detect
            - record
    detect:
      width: 1920
      height: 1080
      fps: 15
    rtmp:  
      enabled: False
```

### Relevant log output

```shell
2023-12-12 16:41:03.720066011  [INFO] Starting Frigate...
2023-12-12 16:41:04.703088334  [2023-12-12 17:41:04] frigate.app                    INFO    : Starting Frigate (0.12.1-367d724)
2023-12-12 16:41:04.711012620  [2023-12-12 17:41:04] frigate.app                    INFO    : Creating directory: /tmp/cache
2023-12-12 16:41:04.711438595  [2023-12-12 17:41:04] peewee_migrate                 INFO    : Starting migrations
2023-12-12 16:41:04.715005985  [2023-12-12 17:41:04] peewee_migrate                 INFO    : There is nothing to migrate
2023-12-12 16:41:04.731897010  [2023-12-12 17:41:04] frigate.app                    INFO    : Output process started: 227
2023-12-12 16:41:04.736387989  [2023-12-12 17:41:04] frigate.app                    INFO    : Camera processor started for Tapo_Camera_Door: 231
2023-12-12 16:41:04.739658122  [2023-12-12 17:41:04] frigate.app                    INFO    : Capture process started for Tapo_Camera_Door: 232
2023-12-12 16:41:04.755459940  [2023-12-12 17:41:04] detector.cpu                   INFO    : Starting detection process: 226
2023-12-12 16:41:04.755699167  [2023-12-12 17:41:04] frigate.detectors              WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
2023-12-12 16:41:05.096902685  [2023-12-12 17:41:05] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:41:05.096906042  [2023-12-12 17:41:05] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:41:24.799534695  [2023-12-12 17:41:24] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:41:24.800124066  [2023-12-12 17:41:24] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:41:24.800430366  [2023-12-12 17:41:24] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x5605628cce00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:41:24.800621174  [2023-12-12 17:41:24] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:41:25.012866168  [2023-12-12 17:41:25] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:41:25.013397608  [2023-12-12 17:41:25] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:41:34.803063555  [2023-12-12 17:41:34] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:41:34.803067284  [2023-12-12 17:41:34] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:41:34.803068803  [2023-12-12 17:41:34] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x55dc34d3ae00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:41:34.803072592  [2023-12-12 17:41:34] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:41:35.021567292  [2023-12-12 17:41:35] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:41:35.021786489  [2023-12-12 17:41:35] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:41:44.810394753  [2023-12-12 17:41:44] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:41:44.810530561  [2023-12-12 17:41:44] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:41:44.810774377  [2023-12-12 17:41:44] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x56100dbf1e00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:41:44.810776670  [2023-12-12 17:41:44] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:41:45.026190486  [2023-12-12 17:41:45] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:41:45.026508364  [2023-12-12 17:41:45] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:41:54.815733806  [2023-12-12 17:41:54] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:41:54.816045125  [2023-12-12 17:41:54] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:41:54.816047357  [2023-12-12 17:41:54] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x556ba124fe00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:41:54.816049040  [2023-12-12 17:41:54] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:41:55.033057373  [2023-12-12 17:41:55] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:41:55.033476502  [2023-12-12 17:41:55] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:42:04.821629099  [2023-12-12 17:42:04] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:42:04.821632893  [2023-12-12 17:42:04] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:42:04.821634466  [2023-12-12 17:42:04] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x556d071f4e00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:42:04.821636099  [2023-12-12 17:42:04] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:42:05.037023477  [2023-12-12 17:42:05] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:42:05.037502032  [2023-12-12 17:42:05] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:42:14.825573586  [2023-12-12 17:42:14] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:42:14.826042925  [2023-12-12 17:42:14] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:42:14.826195032  [2023-12-12 17:42:14] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x55b485054e00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:42:14.826332813  [2023-12-12 17:42:14] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:42:15.045369536  [2023-12-12 17:42:15] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:42:15.045639658  [2023-12-12 17:42:15] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:42:24.834083312  [2023-12-12 17:42:24] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:42:24.834086832  [2023-12-12 17:42:24] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:42:24.834088410  [2023-12-12 17:42:24] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x558382809e00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:42:24.834090078  [2023-12-12 17:42:24] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:42:25.051799427  [2023-12-12 17:42:25] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:42:25.052334607  [2023-12-12 17:42:25] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:42:34.837546932  [2023-12-12 17:42:34] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:42:34.838792478  [2023-12-12 17:42:34] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:42:34.838795619  [2023-12-12 17:42:34] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x562f4fd2ce00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:42:34.838797294  [2023-12-12 17:42:34] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:42:35.052431883  [2023-12-12 17:42:35] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:42:35.052707668  [2023-12-12 17:42:35] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:42:44.845574600  [2023-12-12 17:42:44] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:42:44.845845161  [2023-12-12 17:42:44] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:42:44.845847317  [2023-12-12 17:42:44] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x555a986dfe00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:42:44.845849033  [2023-12-12 17:42:44] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:42:45.058635304  [2023-12-12 17:42:45] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:42:45.059135159  [2023-12-12 17:42:45] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:42:54.851538229  [2023-12-12 17:42:54] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:42:54.851541637  [2023-12-12 17:42:54] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:42:54.851543219  [2023-12-12 17:42:54] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x558f39b7ce00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:42:54.851544816  [2023-12-12 17:42:54] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:42:55.066966743  [2023-12-12 17:42:55] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:42:55.067227831  [2023-12-12 17:42:55] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:43:04.854468530  [2023-12-12 17:43:04] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:43:04.855009938  [2023-12-12 17:43:04] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:43:04.855012662  [2023-12-12 17:43:04] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x556047868e00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:43:04.855014300  [2023-12-12 17:43:04] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:43:05.075612846  [2023-12-12 17:43:05] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:43:05.075933612  [2023-12-12 17:43:05] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:43:14.860392495  [2023-12-12 17:43:14] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:43:14.860913652  [2023-12-12 17:43:14] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:43:14.860916216  [2023-12-12 17:43:14] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x55d43db99e00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:43:14.860917849  [2023-12-12 17:43:14] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:43:15.076400276  [2023-12-12 17:43:15] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:43:15.076720156  [2023-12-12 17:43:15] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:43:24.866009385  [2023-12-12 17:43:24] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:43:24.866542397  [2023-12-12 17:43:24] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:43:24.866892584  [2023-12-12 17:43:24] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x55f3dd482e00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:43:24.866895079  [2023-12-12 17:43:24] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:43:25.082007777  [2023-12-12 17:43:25] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:43:25.082219005  [2023-12-12 17:43:25] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:43:34.871540044  [2023-12-12 17:43:34] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:43:34.872853695  [2023-12-12 17:43:34] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:43:34.872856111  [2023-12-12 17:43:34] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x561db5805e00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:43:34.872857804  [2023-12-12 17:43:34] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:43:35.087248028  [2023-12-12 17:43:35] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:43:35.087474784  [2023-12-12 17:43:35] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:43:44.877416860  [2023-12-12 17:43:44] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:43:44.878862859  [2023-12-12 17:43:44] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:43:44.878865949  [2023-12-12 17:43:44] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x5625533cbe00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:43:44.878867655  [2023-12-12 17:43:44] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:43:45.117693560  [2023-12-12 17:43:45] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:43:45.117954675  [2023-12-12 17:43:45] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:43:54.883482834  [2023-12-12 17:43:54] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:43:54.883486582  [2023-12-12 17:43:54] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:43:54.883488125  [2023-12-12 17:43:54] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x55f463e49e00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:43:54.883489809  [2023-12-12 17:43:54] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:43:55.098133139  [2023-12-12 17:43:55] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:43:55.098393598  [2023-12-12 17:43:55] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:44:04.889186619  [2023-12-12 17:44:04] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:44:04.889703061  [2023-12-12 17:44:04] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:44:04.889706122  [2023-12-12 17:44:04] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x559e4b037e00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:44:04.889707823  [2023-12-12 17:44:04] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:44:05.107191493  [2023-12-12 17:44:05] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:44:05.107455498  [2023-12-12 17:44:05] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
2023-12-12 16:44:14.895029615  [2023-12-12 17:44:14] watchdog.Tapo_Camera_Door      ERROR   : Ffmpeg process crashed unexpectedly for Tapo_Camera_Door.
2023-12-12 16:44:14.895453107  [2023-12-12 17:44:14] watchdog.Tapo_Camera_Door      ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-12-12 16:44:14.895455299  [2023-12-12 17:44:14] ffmpeg.Tapo_Camera_Door.detect ERROR   : [rtsp @ 0x557f2f68de00] method DESCRIBE failed: 401 Unauthorized
2023-12-12 16:44:14.895456950  [2023-12-12 17:44:14] ffmpeg.Tapo_Camera_Door.detect ERROR   : rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)
2023-12-12 16:44:15.117539586  [2023-12-12 17:44:15] frigate.video                  ERROR   : Tapo_Camera_Door: Unable to read frames from ffmpeg process.
2023-12-12 16:44:15.117849871  [2023-12-12 17:44:15] frigate.video                  ERROR   : Tapo_Camera_Door: ffmpeg process is not running. exiting capture thread...
```

### FFprobe output from your camera

```shell
I don't know how or where to run it
```

### Operating system

Debian

### Install method

HassOS Addon

### Coral version

CPU (no coral)

### Network connection

Wireless

### Camera make and model

Tapo C310

**Comment by NickM-27** [comment]
`rtsp://*:*@192.168.0.26/stream2: Server returned 401 Unauthorized (authorization failed)` username or password is wrong

**Comment by MalekBakary** [comment]
No I made them like this *** only here 🙂

**Comment by NickM-27** [comment]
the error 401 unauthorized means that the username or password is wrong according to the camera

**Comment by MalekBakary** [comment]
Thanks it is working now

**Comment by Caddyman68** [comment]
> Thanks it is working now

What did you do to make it work? I have the same problem and KNOW my username and password are correct. I have one Tapo C100 working in Frigate. New one gives the error. Tried setting up new cam with the same credentials as the other, and when I got the error I changed them to unique user/pw. Stream comes up in VLC - copied from there and pasted into Frigate. I'm lost - again

**Comment by MalekBakary** [comment]
In my case I hade symbols in my password, the VLC could read it but not Friget. I simply changed the password to something else

**Comment by Caddyman68** [comment]
Thanks for the reply. Alas that isn't my problem. I have one Tapo up and running in Frigate; thought the new one possibly needed unique credentials so I changed one number in user and also a number in password. Both are pretty short and only include numbers and letters. I'll try to make them short and only lower case letters. Maybe Frigate only allows one Tapo in yml?? - doesn't sound likely or even possible. I updated the firmware in the new one but they are now both on the same version
